src/views/Stack/Card.tsx

```diff
@@ -382,8 +390,16 @@ export default class Card extends React.Component<Props> {
   private handleGestureEventVertical = Animated.event([
     {
       nativeEvent: {
-        translationY: this.gesture,
-        velocityY: this.velocity,
+        translationY: (y: Animated.Adaptable<number>) =>
+          set(
+            this.gesture,
+            multiply(y, cond(this.isVerticalGestureDirectionInverted, -1, 1))
```

Comment:
TRUE = 1, FALSE = 0.
```
multiply(y, this.isVerticalGestureDirectionInverted)
```
should be fine!

Reply:
sorry, no I realised that I'm saying wrong. Could you rather make variable like `verticalGestureDirection` with values LEFT and RIGHT which hold 1 and -1 and apply them while updating?

Reply:
sorry, TOP, BOTTOM

Reply:
Done TOP and BOTTOM created